pkg/compass/compass.go

```diff
@@ -16,7 +16,7 @@ const (
 	osDarwin = "darwin"
 
 	habitatPathEnv         = "HABITAT_PATH"
-	defaultHabitatPathUnix = "~/.habitat"
```

Comment:
I think we should keep the `.`, since it keeps the folder hidden which is fine for people using habitat normally. If someone needs to override, they can just set `HABITAT_PATH` env var